Trading/Exchange/okex_websocket_feed/tests/test_unauthenticated_mocked_feeds.py

```diff
@@ -0,0 +1,46 @@
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+import pytest
+
+import octobot_commons.channels_name as channels_name
+import octobot_commons.enums as commons_enums
+import octobot_commons.tests as commons_tests
+import octobot_trading.exchanges as exchanges
+import octobot_trading.util.test_tools.exchanges_test_tools as exchanges_test_tools
```

Comment:
we should remove this line now, it's unused :)

Reply:
Right :+1: